graph/path/dijkstra.go

```diff
@@ -20,8 +20,30 @@ import (
 //
 // The time complexity of DijkstraFrom is O(|E|.log|V|).
 func DijkstraFrom(u graph.Node, g traverse.Graph) Shortest {
+	return dijkstraFrom(u, nil, g)
+}
+
+// DijkstraFromTo returns a shortest path from u to v in the graph g. The
+// result is equivalent to DijkstraFrom(u, g).To(v), but DijkstraFromTo can be
+// more efficient, as it can terminate early if v is reached. If the graph does
+// not implement Weighted, UniformCost is used. DijkstraFromTo will panic if g
+// has a u-reachable negative edge weight that is discovered before reaching v.
+//
+// The time complexity of DijkstraFromTo is O(|E|.log|V|).
+func DijkstraFromTo(u graph.Node, v graph.Node, g traverse.Graph) (path []graph.Node, weight float64) {
```

Comment:
```suggestion
func DijkstraFromTo(u, t graph.Node, g traverse.Graph) (path []graph.Node, weight float64) {
```
The second param is not a v.